script/Arcueid/Day 1/01_00_ARC01_3.txt

```diff
@@ -177,10 +177,10 @@ I'll―――
 [sha:afc11b33608d21461358eff944b820119d5706a1]{
 -- Page 202, Offset 42575. Choice.
 -- 学校をぶらつく。
- Wander around the school.
+ wander around the school.
```

Comment:
We've generally kept these caps before by the look of it. They both _continue_ the sentence, but I feel like the aesthetics are nicer if they start with a capital since there's several of them.

Reply:
i thought we did but i came across non capitalized ones in the later days. Should i go ahead capitalize those too?
https://github.com/Tsukihimates/Tsukihime-Translation/blob/88c5d839938d0ebe78058ebb3bb56c1d1c33952a/script/Arcueid/Day%206/06_00_ARC06_2_2.txt#L485
https://github.com/Tsukihimates/Tsukihime-Translation/blob/88c5d839938d0ebe78058ebb3bb56c1d1c33952a/script/Arcueid/Day%208/08_00_ARC08_1B.txt#L141
https://github.com/Tsukihimates/Tsukihime-Translation/blob/88c5d839938d0ebe78058ebb3bb56c1d1c33952a/script/Arcueid/Day%208/08_00_ARC08_2B.txt#L332

Reply:
Yeah I'd say standardize on caps if we're gonna standardize.